src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -216,12 +228,7 @@ export class ArticlesTable extends Component {
               title={item.get('title')}
               url={item.get('url')}
               onArticleChanged={({ title, tags }) => { this.props.updateArticle(item.get('id'), { title, tags }); }}
-              onNotesClick={() =>
-                    this.openNotes({
-                      id: item.get('id'),
-                      notes: item.get('notes') ? item.get('notes').toJS() : [],
-                    })
-                  }
+              onNotesClick={() => this.openNotes(i)}
```

Comment:
I would pass `articleId` rather than article index in the table.

Reply:
Article index in the table allows easier/faster article retrieval (which is used on every `render()` when notes are open).

Reply:
Ok, now I think `articleId` is more appropriate if table dynamically changes (e.g., new articles loaded while notes are open)

Reply:
Right.